hw/drivers/flash/spiflash/src/spiflash.c

```diff
@@ -722,15 +753,21 @@ uint8_t
 spiflash_read_status(struct spiflash_dev *dev)
 {
     uint8_t val;
+    const uint8_t cmd = SPIFLASH_READ_STATUS_REGISTER;
 
     spiflash_lock(dev);
```

Comment:
do we still need this lock for bus driver?

Reply:
lock removed for bus driver